.circleci/config.yml

```diff
@@ -471,6 +471,48 @@ jobs:
           file: build/logs/clover.xml
       - save-npm-cache
 
+  behat-absolute-url-coverage:
```

Comment:
Is it really necessary to create a new environment to test this config option?

Reply:
I don't know :). Do you have any recommendations? I assumed I needed to because it is a "global" configuration. Is there a way I can load certain configuration options for specific tests?

Reply:
I think that because of behat we're a bit limited over what we can configure, this looks fine to me. @teohhanhui maybe you have improvement thoughts on this?

Reply:
Any idea why the new environment is failing? 
```
#!/bin/bash -eo pipefail
mkdir -p "$(dirname -- "build/logs/clover.xml")"
phpcov merge --clover "build/logs/clover.xml" "build/coverage"
```
is giving the error 
```
The "build/coverage" directory does not exist.
```
but this is reusing the same configuration as the rest of the circleci tests.